bin/deploy-hobby

```diff
@@ -4,6 +4,7 @@ set -e
 
 export POSTHOG_APP_TAG="${POSTHOG_APP_TAG:-latest}"
 export SENTRY_DSN="${SENTRY_DSN:-'https://public@sentry.example.com/1'}"
+export SKIP_ASYNC_MIGRATIONS_SETUP="${SKIP_ASYNC_MIGRATIONS_SETUP:-0}"
```

Comment:
these envvars are only used when templating the final docker compose. We could indeed make it configurable and source it in the compose, but I think it's fine to just hardcode it to false in [this list of envvars](https://github.com/PostHog/posthog/blob/9e5b92e7d8ff4bbc76e5ba589d253fd06f70a7a4/docker-compose.base.yml#L68) that the django containers inherit.

Reply:
oh 🤦‍♀️  good catch